packages/moonstone/Dropdown/Dropdown.js

```diff
@@ -275,7 +279,8 @@ const DropdownBase = kind({
 	},
 
 	render: ({children, disabled, onOpen, onSelect, open, selected, width, title, ...rest}) => {
-		const popupProps = {children, onSelect, selected, width};
+		const role = '';
```

Comment:
Is the intent here to pre-define a default?
```
		const popupProps = {children, onSelect, selected, width, role: ''};
```

Reply:
Nope :) Changed